### PR TITLE
Feature/error handling

### PR DESCRIPTION
**Pull request by tonycarpenter21**
What (if any) features are you implementing?
- Error Handling

What (if anything) did you refactor?
- Styling

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: src/404/404.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const ErrorHandling = () => {
+  return (
+    <section className="error">
+      <img src="https://media2.giphy.com/media/rYEAkYihZsyWs/giphy.gif?cid=ecf05e47mxw0p2ev880va40mrexkbc8mm0gvmprvp8elek15&rid=giphy.gif&ct=g" alt="Bob Ross 404 error"/>
+      <h1 className="error-message">Page not found. Let's just paint a happy little 404 message right here!</h1>
+    </section>
+  )
+}
+
+export default ErrorHandling;
```
Comment (mr-ryan12) [inline src/404/404.tsx:13]: Thank you for making this! Looks great!

File: src/App.tsx [hunk 2]
```diff
@@ -19,13 +20,24 @@ class App extends Component<{}, State> {
...
         <main className="App">
-          < Route exact path='/' render={() => <Categories categories={this.state.categories}/> }/>
-          < Route exact path='/:category' render={({ match }) => <ArtView category={match.params.category}/>} />
+          <Switch>
+            < Route exact path='/' render={() => <Categories categories={this.state.categories}/> }/>
+            < Route exact path='/:category' render={({ match }) => this.checkCategory(match.params.category) } />
+            < Route render={() => <ErrorHandling /> } />
+          </Switch>
         </main>
       </>
```
Comment (mr-ryan12) [inline src/App.tsx:42]: Glad we got this figured out! Nice work!

File: src/apiCalls.tsx [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+        throw new Error(`${response.status}. I am so sorry for the inconvenience! The server is down. Please try again, later.`)
+      } else if (response.status === 404) {
+        throw new Error(`${response.status}. We apologize for the inconvenience. Please try again, or go back to the Main Page.`)
+      } else {
+        throw new Error(`${response.status}. We apologize for the inconvenience. Please try again later.`)
+      }
+    } else {
+      return response.json()
+    }
+  })
```
Comment (mr-ryan12) [inline src/apiCalls.tsx:15]: Thank you for making the error messages dynamic based on the status code!

**Review by Jayquintana — APPROVED** [review]
good stuff tony !

**Pull request merged**